bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/PsbtRpcTest.scala

```diff
@@ -124,4 +124,16 @@ class PsbtRpcTest extends BitcoindRpcTest {
       assert(result.issolvable.equals(true))
     }
   }
+
+  it should "get node address" in {
+    val startF = client.start()
+    val nodeF = startF.flatMap(_ => client.getNodeAddresses())
+
+    //Todo figure out how to check time and port

```

Comment:
Its not important to test every field of the return type. If it parses correctly its most likely correct:-)